install-and-configure/install/multi-cluster/federated-etl/thanos-migration-guide.md

```diff
@@ -156,11 +161,7 @@ kubecostModel:
 
 Optionally, you can remove the [Thanos sidecar](https://raw.githubusercontent.com/kubecost/cost-analyzer-helm-chart/v1.108.1/cost-analyzer/values-thanos.yaml) running on this secondary cluster. If left on, this secondary cluster will continue to push Prometheus metrics to the object store which can be used as a backup.
 
-### Step 8: Remove Thanos configuration from the primary cluster
-
-Remove [Thanos configurations](https://raw.githubusercontent.com/kubecost/cost-analyzer-helm-chart/v1.108.1/cost-analyzer/values-thanos.yaml) from your primary cluster since it is no longer included in the Helm chart in 2.0. Your Kubecost installation is now configured to query the Aggregator for multi-cluster data, instead of Thanos.
-
-### Step 9: Upgrade primary cluster to v2.0
+### Step 9: Upgrade primary cluster to v2.x (2.3+ recommended)
```

Comment:
```suggestion
### Step 9: Upgrade primary cluster to v2.x
```

I'm removing (2.3+ recommended) because that will likely become outdated quickly.